ravenpy/models/emulators.py

```diff
@@ -641,6 +659,226 @@ def ost2raven(self, ops):
         return self.params(*out)
 
 
+class CANADIANSHIELD(Raven):
+    """
+    The Canadian Shield model is a useful configuration of Raven for Canadian shield basins characterized by shallow soils atop rock, with ample exposed rock and lakes.
+
+    References
+    ----------
+    None
+    """
+
+    identifier = "canadianshield"
+    templates = tuple((Path(__file__).parent / "raven-canadianshield").glob("*.rv?"))
+
+    params = namedtuple(
+        "CANADIANSHIELDParams",
+        ", ".join(["par_x{:02}".format(i) for i in range(1, 35)]),
+    )
+
+    @dataclass
+    class HRU_ORGANIC(HRU):
+        hru_id: int = 1
+        land_use_class: str = "FOREST"
+        veg_class: str = "FOREST"
+        soil_profile: str = "SOILP_ORG"
+        aquifer_profile: str = "[NONE]"
+        terrain_class: str = "[NONE]"
+
+    @dataclass
+    class HRU_BEDROCK(HRU):
+        hru_id: int = 2
+        land_use_class: str = "FOREST"
+        veg_class: str = "FOREST"
+        soil_profile: str = "SOILP_BEDROCK"
+        aquifer_profile: str = "[NONE]"
+        terrain_class: str = "[NONE]"
+
+    def __init__(self, *args, **kwds):
+        super().__init__(*args, **kwds)
+        self.rvp = RVP(
+            params=CANADIANSHIELD.params(
+                *((None,) * len(CANADIANSHIELD.params._fields))
+            ),
+            # TODO:: use here LU set by user in test_candianshield
+            land_use_classes=(
+                LU("FOREST", impermeable_frac=0.0, forest_coverage=0.02345),
+            ),
+        )
```

Comment:
The `LU` should not be hard-coded here.

Reply:
See this rather as the default value, not something that is hard-coded. In principle, the values you pass to the model at execution should override this.

Reply:
If you're saying that the LU given in the test does not override the default value given in the CANADIANSHIELD class, then this is a bug for me to fix.

Reply:
I haven't actually tested if it would be overwritten when I initiate it with a different value. Let me check.

Reply:
Of course it's not a bug :) 

It works as you said. This is overwritten by whatever I pass to the model at execution. I think my logic was just exactly the other way around. So, it's all good.